concent_api/middleman/tests/test_queue_pool.py

```diff
@@ -0,0 +1,119 @@
+from asyncio import Queue
+from logging import Logger
+import pytest
+from assertpy import assert_that
+from freezegun import freeze_time
+from mock import create_autospec
+from mock import patch
+from mock import sentinel
+
+from golem_messages.message import Ping
+
+from common.helpers import get_current_utc_timestamp
+from middleman.constants import ResponseQueueItem
+from middleman.utils import QueuePool
+
+
+class TestQueuePoolInitialization:  # pylint: disable=no-self-use
+    def test_that_queue_pool_is_created_with_given_params(self, event_loop):
```

Comment:
Tests should have typings and return type too.

Reply:
typing in tests is a bit of overkill... I could add it, but I think we should introduce it to our coding style if we want it to be a common practice...

Reply:
I am currently adding them to all tests in https://github.com/golemfactory/concent/issues/103 so once we turn on those options in mypy those places which miss them will produce errors.

Reply:
tests should be excluded from mypy checks. Adding mypy to tests increases the maintenance costs of tests. And I have never encountered any project that would enforce type annotations in unit tests.
If we want to introduce it, it **must** be discussed first.